One Agent/README.md

```diff
@@ -0,0 +1,23 @@
+# One Agent Deploy If not Exist
+
+One agent deployed agents on your virtual machines. This enables you to recreate activity trails for investigation purposes when a security incident occurs or your network is compromised. [need to edit]
```

Comment:
Do you want to remove the [need to edit] tag?

Reply:
@tarosler - Ben Kliger will update the text so it just temporary tab / text.